### ISSUE TITLE
How to install previous versions of this package?

### ISSUE THREAD

**Issue by ElsLommelen** [inbo/checklist#61]
I tried finding out how to install the previous version of the checklist package (`0.1.11`), but it seems to me that only versions up to `0.1.7` are available as releases.  I tried guessing the tag, but this gave an error:
``` r
remotes::install_github("inbo/checklist@v0.1.11")
#> Downloading GitHub repo inbo/checklist@v0.1.11
#> Error in utils::download.file(url, path, method = method, quiet = quiet,  : 
#>   cannot open URL 'https://api.github.com/repos/inbo/checklist/tarball/v0.1.11'
```

<sup>Created on 2021-07-27 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

After installing the INBO universe and successfully installing the newest version with `install.packages()` for verification, a 'regular' installation of the previous version also gave an error:

``` r
remotes::install_version("checklist", version = "0.1.11")
#> Trying https://inbo.r-universe.dev
#> Trying https://cloud.r-project.org
#> Error in download_version_url(package, version, repos, type): version '0.1.11' is invalid for package 'checklist'
```

<sup>Created on 2021-07-27 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

Also the [website with releases](https://api.github.com/repos/inbo/checklist/releases) only seems to contain releases up to 0.1.7, so I guess your attempt to produce releases automatically, is not working properly?  In case these previous versions can be installed another way, it would be good to mention it in the installation instructions.  (I could of course checkout the right git commit and install it this way, but it is rather complicated to script this for non-local use...)

**Comment by ElsLommelen** [comment]
A related issue: the [website ](https://inbo.github.io/checklist/index.html)mentions version 0.1.11 on top and [news ](https://inbo.github.io/checklist/news/index.html)is updated to version 0.1.11, but when installing the newest version by using `remotes::install_github("inbo/checklist")`, I get version 0.1.12 installed.  As [this PR](https://github.com/inbo/checklist/pull/54) is merged to the main, I suppose version 0.1.12 is the actual version, but it seems the website is not updated.  So in the above comment I wanted to return to 0.1.11 from 0.1.12.

**Comment by ThierryO** [comment]
I'm having problems to set the tags automatically to the versions. If you need a specific version, then find the commit hash of that version. Then you can use `remotes::install_github("inbo/checklist@place_here_commit_sha")`.
